services/QuillLMS/app/views/pages/home_new.html.erb

```diff
@@ -62,7 +62,7 @@
 <section class='tool-section bg-tool-orange'>
   <div class="tool connect-tool">
     <div class='info-section'>
-      <a href="/tools/connect/"><h2 class='q-h2'><img class="icon-white lazyload" data-src='https://assets.quill.org/images/shared/quill_connect_white.svg'>Quill Connect</h2></a>
+      <a href="/tools/connect/"><h2 class='q-h2'><img class="icon-white lazyload" data-src='https://assets.quill.org/images/shared/quill_connect_white.svg' alt="Quill Connect Icon">Quill Connect</h2></a>
```

Comment:
I think we might want to use blank alt tags for these (which tell the screenreader the images are just decorative) instead of naming the tool. The description isn't really meaningful to anyone who doesn't already know what that icon is, and we read the name of the tool right after so a screenreader user will hear "Quill Connect Icon Quill Connect", which seems annoying.